provider/go.mod

```diff
@@ -17,8 +17,8 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pulumi/providertest v0.1.3
 	github.com/pulumi/pulumi-terraform-bridge/v3 v3.98.0
-	github.com/pulumi/pulumi/pkg/v3 v3.143.0
-	github.com/pulumi/pulumi/sdk/v3 v3.143.0
+	github.com/pulumi/pulumi/pkg/v3 v3.144.1
```

Comment:
We currently prefer a slightly non-standard upgrade path for bridged providers where these dependencies are pulled in via bridge releases only, which has some test qualification benefits for us, and subsequently this is managed by upgrade provider. 

There should be a config for Renovate to exclude these particular dependencies? @blampe